wordpress-mobile/WordPress-iOS
# Update CircleCI for Xcode 11

This PR fixes CircleCI with Xcode 11 (it used to be for re-enabling ZenDesk, but there was a lot of other stuff required for that)

**To test:**
- Checkout the branch and install pods
- Build and run (this will ensure that the pods installed right)
- Ensure that tests pass (this will ensure that we have CircleCI set up for XC11)

**Update release notes:**
